cmd/scollector/collectors/program.go

```diff
@@ -151,12 +178,20 @@ Loop:
 			Value:     val,
 			Tags:      opentsdb.TagSet{"host": util.Hostname},
 		}
-		for _, tag := range sp[3:] {
-			tags, err := opentsdb.ParseTags(tag)
+		for _, field := range sp[3:] {
+			if strings.HasPrefix(field, "meta.") {
```

Comment:
I like the idea of a separate line for metadata. Works better with grep when debugging.